heareval/tasks/pipeline.py

```diff
@@ -239,6 +265,13 @@ def run(self):
             index=False,
         )
 
+        for split, split_df in process_metadata.groupby("split"):
```

Comment:
What is this doing? Is it redundant?

Reply:
It is grouping by the split to see count for each group